sql/postgres/inspect.go

```diff
@@ -772,6 +772,16 @@ type (
 		Columns []*schema.Column
 	}
 
+	// IndexBuildParams describes additional parameters for index creation.
+	IndexBuildParams struct {
+		schema.Attr
+
+		// Concurrently describes the CONCURRENTLY clause to instruct Postgres
+		// to build the index concurrently without blocking the current table.
+		// https://www.postgresql.org/docs/current/sql-createindex.html#SQL-CREATEINDEX-CONCURRENTLY
+		Concurrently bool
+	}
```

Comment:
I'm thinking, maybe we can name the attribute as `Concurrently` to allow using it in other schema changes like `schema.DropIndex` (https://www.postgresql.org/docs/current/sql-dropindex.html).

Reply:
Done. I added support for `DROP INDEX CONCURRENTLY` as well.